source/brailleDisplayDrivers/eurobraille.py

```diff
@@ -284,51 +301,51 @@ def _handleAck(self, frame):
 			except KeyError:
 				log.debugWarning("Received ACK for unregistered frame %d"%frame)
 
-	def _handleSystemPacket(self, type, data):
-		if type==EB_SYSTEM_TYPE:
+	def _handleSystemPacket(self, packetType: bytes, data: bytes):
+		if packetType == EB_SYSTEM_TYPE:
 			deviceType = ord(data)
 			self.deviceType = DEVICE_TYPES[deviceType]
-			if 0x01<=deviceType<=0x06: # Iris
+			if 0x01 <= deviceType <= 0x06:  # Iris
 				self.keys = KEYS_IRIS
-			elif 0x07<=deviceType<=0x0d: # Esys
+			elif 0x07 <= deviceType <= 0x0d:  # Esys
 				self.keys = KEYS_ESYS
-			elif 0x0e<=deviceType<=0x11: # Esitime
+			elif 0x0e <= deviceType <= 0x11:  # Esitime
 				self.keys = KEYS_ESITIME
 			else:
 				log.debugWarning("Unknown device identifier %r"%data)
-		elif type==EB_SYSTEM_DISPLAY_LENGTH:
+		elif packetType == EB_SYSTEM_DISPLAY_LENGTH:
 			self.numCells = ord(data)
-		elif type==EB_SYSTEM_FRAME_LENGTH:
+		elif packetType == EB_SYSTEM_FRAME_LENGTH:
 			self._frameLength = bytesToInt(data)
-		elif type==EB_SYSTEM_PROTOCOL and self.isHid:
-			protocol = data.rstrip("\x00 ")
+		elif packetType == EB_SYSTEM_PROTOCOL and self.isHid:
+			protocol = data.rstrip(b"\x00 ")
 			try:
 				version = float(protocol[:4])
 			except ValueError:
 				pass
 			else:
-				self.receivesAckPackets = version>=3.0
-		elif type==EB_SYSTEM_IDENTITY:
-			return # End of system information
-		self._deviceData[type]=data.rstrip("\x00 ")
+				self.receivesAckPackets = version >= 3.0
+		elif packetType == EB_SYSTEM_IDENTITY:
+			return  # End of system information
+		self._deviceData[packetType] = data.rstrip(b"\x00 ")
 
-	def _handleKeyPacket(self, group, data):
+	def _handleKeyPacket(self, group: bytes, data: bytes):
 		if group == EB_KEY_USB_HID_MODE:
-			self._hidKeyboardInput = bool(int(data))
+			self._hidKeyboardInput = boolToBytes(data)

```

Comment:
I don't think this does the same. bool(int(data)) will be False if data == b"0", where byte2bool will use the ordinal of the byte to check the boolean value. See my comment above regarding the eurobraille protocol using real numbers for boolean values.